examples/feature_selection/plot_feature_selection_pipeline.py

```diff
@@ -32,3 +33,8 @@
 anova_svm.fit(X_train, y_train)
 y_pred = anova_svm.predict(X_test)
 print(classification_report(y_test, y_pred))
+
+# access and plot the coefficients of the fitted model
+plt.bar((0, 1, 2), anova_svm.named_steps.svc.coef_.ravel())
+plt.xticks((0, 1, 2), anova_svm.named_steps.svc.input_features_)
+plt.show()
```

Comment:
Personally I prefer `barh` to make feature names more readable (horizontal).

Reply:
+1 (pet personal peeve :D )

Reply:
I actually prefer hbar as well ;)